src/mode/sql_highlight_rules.js

```diff
@@ -22,7 +28,7 @@ var SqlHighlightRules = function() {
 
     var dataTypes = (
         "int|numeric|decimal|date|varchar|char|bigint|float|double|bit|binary|text|set|timestamp|" +
-        "money|real|number|integer"
+        "money|real|number|integer|string"
```

Comment:
is string part of the ANSI specifications? or only in duckdb?

Reply:
Its not in the spec but very common